manubot/manubot.py

```diff
@@ -194,6 +195,12 @@ def add_author_affiliations(variables):
         if 'affiliations' not in author:
             continue
         if not isinstance(author['affiliations'], list):
+            warnings.warn(
```

Comment:
Will this warning appear if an author has a single affiliation?

Reply:
If there affiliations are a list with one affiliation it will not occur.

Reply:
Thanks.  I wasn't sure how the YAML would be parsed and whether the affiliation would be a string or a single element list.